backend/variables/builtin/custom-variable-keys.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const customVariableManager = require("../../common/custom-variable-manager");
+
+const { OutputDataType, VariableCategory } = require("../../../shared/variable-constants");
+
+function isObject(data) {
+    return typeof data === 'object' && !(typeof data === 'string' || data instanceof String);
```

Comment:
Can be simplified to:

```js
return typeof data === 'object' && !(data instanceof String)
```

If `data` returns truthy against the `typeof data === 'object'` check, it won't ever pass the `=== 'string'` check